plugins/tooltip-old.js

```diff
@@ -0,0 +1,414 @@
+(function (factory) {
```

Comment:
Sure. I was going to do it before release. Now I want to save it as a quick reference if any regression bugs appear